Remotion/Web/ClientScript/Scripts/ViewLayout.ts

```diff
@@ -16,47 +16,63 @@
 // 
 class ViewLayout
 {
-  public static AdjustActiveViewContent (viewContent: JQuery): void
+  public static AdjustActiveViewContent (viewContent: HTMLElement): void
   {
-    var viewContentBorder = viewContent.children('div').eq (0);
-    var viewBottomControls = viewContent.next();
+    var viewContentBorder = viewContent.querySelector ('div')!;
```

Comment:
`.children()` requires a `:scope >`

```suggestion
    var viewContentBorder = viewContent.querySelector (':scope > div')!;
```